code/game/mecha/equipment/tools/sec_tools.dm

```diff
@@ -108,25 +97,33 @@
 		return 1
 	return
 
-/obj/item/mecha_parts/mecha_equipment/tool/jail/proc/go_out(var/list/L)
-	var/mob/living/ejected = L["mob"]
+/obj/item/mecha_parts/mecha_equipment/tool/jail/proc/go_out(var/L)
+	var/mob/living/ejected = cells[L]
 	if(!ejected)
 		return
 	ejected.forceMove(get_turf(src))
 	occupant_message("[ejected] ejected.")
 	log_message("[ejected] ejected.")
-	L["timer"] = 0
 	ejected.reset_view()
-	/*
-	if(occupant.client)
-	occupant.client.eye = occupant.client.mob
-	occupant.client.perspective = MOB_PERSPECTIVE
-	*/
-	L["mob"] = null
+	cells[L] = null
 	ejected = null
+	if(CellFree())
+		set_ready_state(1)
 	if(AllFree())
 		pr_mech_jail.stop()
-		set_ready_state(1)
+	return 1
+
+/obj/item/mecha_parts/mecha_equipment/tool/jail/proc/subdue(var/L)
+	var/mob/living/prisoner = cells[L]
+	if(!prisoner)
+		return
+	prisoner.Stun(10)
+	prisoner.Knockdown(10)
+	prisoner.apply_effect(STUTTER, 10)
+	chassis.use_power(energy_drain)
```

Comment:
The amount used by using the jail cell itself, 20. Which is coincidentally the same amount used by firing a mounted taser.